page_objects/page_ovirt_hostedengine.py

```diff
@@ -157,11 +157,15 @@ def clean_nfs_storage(self, nfs_ip, nfs_pass, nfs_path):
 
     def move_failed_setup_log(self):
         cmd = "find /var/log -type f |grep ovirt-hosted-engine-setup-.*.log"
-        self.host.execute(cmd)
         if not os.path.exists("/var/old_failed_setup_log"):
```

Comment:
It's meaningless to check whether the directory exists on the machine running testing program. Should to check on the host.